app/controllers/catalog_controller.rb

```diff
@@ -85,6 +85,16 @@ class CatalogController < ApplicationController
     }
 
     config.add_facet_field 'dc_rights_s', label: 'Access', limit: 8, partial: 'icon_facet', show: false
```

Comment:
Didn't see that there. You're right. The field will probably have other values in the future, but not right now.